src/main/drivers/pwm_output.c

```diff
@@ -316,15 +310,14 @@ void motorDevInit(const motorDevConfig_t *motorConfig, uint16_t idlePulse, uint8
         IOConfigGPIO(motors[motorIndex].io, IOCFG_AF_PP);
 #endif
 
-        /* standard PWM outputs */
-        if (useUnsyncedPwm) {
-            pwmRate = motorConfig->motorPwmRate;
-        }
+        /* standard PWM outputs - round up to the nearest 10 for pwmRateHz when not using configured amount */
+        const unsigned pwmRateHz = useUnsyncedPwm ? motorConfig->motorPwmRate : ceilf(1e5f / (usMin + usLen)) * 10;
```

Comment:
I think that this will be fine?
`const unsigned pwmRateHz = useUnsyncedPwm ? motorConfig->motorPwmRate : ceilf(1e6 / (usMin + usLen));`